src/scheduler.h

```diff
@@ -45,14 +45,14 @@ class CScheduler
     void schedule(Function f, boost::chrono::system_clock::time_point t);
 
     // Convenience method: call f once deltaSeconds from now
-    void scheduleFromNow(Function f, int64_t deltaSeconds);
+    void scheduleFromNow(Function f, int64_t deltaMilliSeconds);
 
     // Another convenience method: call f approximately
     // every deltaSeconds forever, starting deltaSeconds from now.
     // To be more precise: every time f is finished, it
     // is rescheduled to run deltaSeconds later. If you
     // need more accurate scheduling, don't use this method.
-    void scheduleEvery(Function f, int64_t deltaSeconds);
+    void scheduleEvery(Function f, int64_t deltaMilliSeconds);
```

Comment:
nit: comments need to be updated (they still mention the old argument `deltaSeconds`).

Reply:
done, squashed into the last commit.